### PR TITLE
Migrated test_gfid_heal

### PR DESCRIPTION
**Pull request by aujjwal-redhat**
Migrated the [test
case](https://github.com/gluster/glusto-tests/blob/HEAD/tests/functional/afr/test_gfid_heal.py)

Updates: #292

Signed-off-by: Ayush Ujjwal <aujjwal@redhat.com>

### CODE REVIEW

**Comment by aujjwal-redhat** [comment]
```sh

✔ Running test case : test_gfid_heal-rep
✔ test_gfid_heal-rep Succeeded
Results for afr
Test nature : disruptive
TestName : test_gfid_heal
+-------------+-------------+-----------------------+-------------+
| Volume Type | Test Result | Time Taken (hh:mm:ss) | Skip Reason |
+-------------+-------------+-----------------------+-------------+
|     rep     |     PASS    |        00:02:38       |      NA     |
+-------------+-------------+-----------------------+-------------+
```

File: tests/functional/afr/test_gfid_heal.py [hunk 1]
```diff
@@ -0,0 +1,110 @@
...
+            if not redant.create_file(f"{brick_path}/dir{i}/", f"file{i}",
+                                      brick_node):
+                raise Exception(f"File creation failed on {brick_node}")
+
+        # To circumvent is_fresh_file() check in glusterfs code.
+        time.sleep(2)
+
+        # Access files from mount
+        for i in range(1, 4):
+            cmd = f"ls {self.mounts[0]['mountpath']}/dir{i}/file{i}"
```
Comment (nik-redhat) [inline tests/functional/afr/test_gfid_heal.py:89]: If we only need to run on one of the clients then why fetch the whole list from the volds, we can simple use `self.mountpoint` and `self.client_list[0]`.

Now, this is something I know is wrong in many places, but I have started to use it accordingly and would suggest you as well.

Comment (aujjwal-redhat) [reply tests/functional/afr/test_gfid_heal.py:89]: Ack

**Comment by aujjwal-redhat** [comment]
```sh

✔ test_gfid_heal-rep Succeeded
Results for afr
Test nature : disruptive
TestName : test_gfid_heal
+-------------+-------------+-----------------------+-------------+
| Volume Type | Test Result | Time Taken (hh:mm:ss) | Skip Reason |
+-------------+-------------+-----------------------+-------------+
|     rep     |     PASS    |        00:02:38       |      NA     |
+-------------+-------------+-----------------------+-------------+
```

**Review by nik-redhat — APPROVED** [review]

**Pull request merged**